indexing-hadoop/src/main/java/org/apache/druid/indexer/InputRowSerde.java

```diff
@@ -340,22 +340,25 @@ public static SerializeResult toBytes(
             parseExceptionMessages.add(e.getMessage());
           }
 
-          String t = aggFactory.getTypeName();
+          final ValueType type = aggFactory.getType();
+          final String typeName = aggFactory.getTypeName();
```

Comment:
nit: these can be  moved to else clause. also `aggFactory.getTypeName()` can just be inlined for complex clause.

Reply:
Ah, good point, will move :+1: